prime-router/examples/generate-jwt-python/README.md

```diff
@@ -7,29 +7,70 @@ This directory contains a python example of how to connect to ReportStream as a
 #### Prerequisites
 
 - Python 3
-- PyJWT module (`python3 -m pip install PyJWT`)
-- cryptography moddule (`python3 -m pip install cryptography`)
+- pipx python package manager
+- Python packages `PyJWT` and `cryptography`
 
 #### Setup
 
-1. Working in this folder, follow the setup steps in
-       https://auth0.com/blog/how-to-handle-jwt-in-python/
-prior to running generate-jwt.py
-
-2. Most of those steps you only need do once. Once you have followed those steps, then each subsequent time you need to run it, just do:
-       `source .env/bin/activate`
-
-3. Then in `generate-jwt.py`, make these two changes:
-- Change `my_client_id` to be your unique client-id string, as given to you by the ReportStream team back in Step 1.
-- Point the `my_rsa_keypair_file` to be the path to your generated rsa keypair pem file.
+1. Install pipx using [their instructions](https://pypa.github.io/pipx/installation/)
+       
+2. Ensure pipx has been added to the PATH
+    <details>
+      <summary>Windows</summary>
+    
+      ```bash
+      python3.exe -m pipx ensurepath
+      ```
+    </details>
+
+    <details>
+      <summary>Linux or Mac</summary>
+    
+      ```bash
+      python3 -m pipx ensurepath
+      ```
+    </details>
+
+3. Install the python dependencies using pipx
+    <details>
+      <summary>Windows</summary>
+    
+      ```bash
+      python3.exe -m pipx install PyJWT cryptography
```

Comment:
This didn't work for me.  I hid two errors:

- it looks like pipx didn't like passing multiple options to install, so at first the command didn't run at all
- I was able to run `python3.exe -m pipx install PyJWT`, but `python3.exe -m pipx install cryptography` did not work, saying there was no associated package